api/data_explorer/controllers/facets_controller.py

```diff
@@ -86,9 +85,12 @@ def facets_get(filter=None, extraFacets=None):  # noqa: E501
     #    'Elasticsearch response: %s' % pprint.pformat(es_response_facets))
     facets = []
     for es_field_name, facet_info in combined_facets.iteritems():
+        current_app.logger.info(
+            'FIELD: %s INFO: %s' % (es_field_name, facet_info))
         name = facet_info.get('ui_facet_name')
         description = facet_info.get('description')
         es_facet = facet_info.get('es_facet')
+        field_type = facet_info.get('type', 'text')
```

Comment:
Remove `text`. It's a bug if facet_info doesn't have `type`.

Reply:
Done.